examples/plot_mne_example.py

```diff
@@ -0,0 +1,223 @@
+# and analyze it as continuous (non-epoched) data.
+#
+
+###################################################################################################
+
+# Get the data path for the MNE example data
+raw_fname = sample.data_path() + '/MEG/sample/sample_audvis_filt-0-40_raw.fif'
+event_fname = sample.data_path() + '/MEG/sample/sample_audvis_filt-0-40_raw-eve.fif'
+
+# Load the file of example MNE data
+raw = mne.io.read_raw_fif(raw_fname, preload=True)
+
+###################################################################################################
+
+# Select EEG channels from the dataset
+raw = raw.pick_types(meg=False, eeg=True, eog=False, exclude='bads')
+
+###################################################################################################
+
+# Explicitly adding a default EEG average reference required for source localization
+raw.set_eeg_reference()
+
+###################################################################################################
+
+# Explore the data
+raw.info()
+
+###################################################################################################
+
+# Plot the channel locations and labels
+raw.plot_sensors(show_names=True)
+
+###################################################################################################
+
+# Creating epochs
+reject = dict(eeg=180e-6)
+event_id, tmin, tmax = {'left/auditory': 1}, -0.2, 0.5
+events = mne.read_events(event_fname)
+epochs = mne.Epochs(raw, events=events, event_id=event_id,
+				tmin=5, tmax=125, baseline=None, preload=True)
+
+###################################################################################################
+
+# Creating Power Spectra Densities
+psd, freqs = mne.time_frequency.psd_welch(epochs, fmin=1., fmax=50., n_fft=2000, n_overlap=250, n_per_seg=500)
+
+###################################################################################################
+# fooofgroup
+# ------------------
+#
+# The FOOOFGroup object is used to fit
+# FOOOF models across the power spectra. A list of FOOOFGroup objects is returned.
+#
+
+###################################################################################################
+
+# Initialize a FOOOF group object, with desired settings
+fg = FOOOFGroup(peak_width_limits=[1, 6], min_peak_amplitude=0.075,
+                    max_n_peaks=6, peak_threshold=1, verbose=False)
+
+###################################################################################################
+
+# Shaping the PSDs so that we can run fooofgroup
+fooof_psd = np.squeeze(psd[4,:,:]) 
+n_channels, n_freq = fooof_psd.shape
+num_blocks = len(mne.read_events(event_fname))
+
+# Setting frequency range
+freq_range = [3, 32]
+
+# This returns a list of FOOOFGRoup objects
+fg.fit(freqs, fooof_psd, freq_range)
+fg.plot()
+
+###################################################################################################
+
+# Define bands of interest
+bands = {'theta': [2,7],
```

Comment:
This band definition isn't entirely consistent: 2 is below the FOOOF fit range above. Also, alpha is more commonly 7-14 Hz, so I'd do: theta:3-7; alpha: 7-14 & beta:15-30.